packages/core/test/perf/runners.js

```diff
@@ -65,6 +66,29 @@ function runSuite(suite) {
     .run();
 }
 
+function mostFromArray (a) {
+  return new FromArray(a)
+}
+
+const PropagateTask = require('../../src/scheduler/PropagateTask').PropagateTask
+const asap = require('../../../scheduler').asap
+
+function FromArray (a) {
+  this.array = a
+}
+
+FromArray.prototype.run = function (sink, scheduler) {
+  return asap(new PropagateTask(runProducer, this.array, sink), scheduler)
```

Comment:
It's sufficient for the zip test, but we should write a custom FromArrayTask for the perf tests instead.  This current hacky approach will not work for anything that uses take or otherwise stops early.